src/traitsHelper.js

```diff
@@ -0,0 +1,6 @@
+export function setValue(component, traitName, newValue) {
+    const trait = component.getTrait(traitName)
+    //const traitOptions = [{value: newValue}]
+    //trait.set('options', [...trait.get('options'), ...traitOptions])
```

Comment:
Is this still needed here?

Reply:
traitsHelper is not actually used right now, but it will be used in the future. I can take out the file for now though.